src/ParaTest/Runners/PHPUnit/SuiteLoader.php

```diff
@@ -214,29 +214,96 @@ private function executableTests($path, $classMethods)
     /**
      * Identify method dependencies, and group dependents and dependees on a single methodBatch
      * If no dependencies exist each methodBatch will contain a single method.
-     * @param  array of ParsedFunction $classMethods
+     * @param  string      $path
+     * @param  ParsedClass $class
      * @return array of MethodBatches. Each MethodBatch has an array of method names
      */
-    private function getMethodBatches($classMethods)
+    private function getMethodBatches($path, $class)
     {
+        $classMethods = $class->getMethods($this->options ? $this->options->annotations : array());
+        $maxBatchSize = $this->options && $this->options->functional ? $this->options->maxBatchSize : 0;
         $methodBatches = array();
         foreach ($classMethods as $method) {
+            $tests = $this->getMethodTests($path, $class, $method);
+            if (!$tests) {
```

Comment:
Is there a way to test this? If someone in the future removes this line (by not knowing what it is for or by accident) nobody will recognize.